### PR TITLE
LOB name in nav, page title

### PR DESCRIPTION
**Pull request by pkarman**
connects #9228 
connects #9116 

### Description
Fixing #9116 in order to test #9228 

Adds the line of business name to the nav links and page title.

### CODE REVIEW

File: spec/models/end_product_spec.rb [hunk 1]
```diff
@@ -291,6 +292,21 @@
         last_action_date: Date.new(2019, 1, 29)
       )
     end
+
+    context "datetime returned in 8601 format" do
+      let(:claim_receive_date) { (Time.zone.now + 10.days).iso8601.to_s }
```
Comment (leikkisa) [inline spec/models/end_product_spec.rb]: Weird that we're getting it in two different formats! Good fix.

File: client/app/nonComp/index.jsx [hunk 2]
```diff
@@ -31,20 +31,20 @@ class NonComp extends React.PureComponent {
...
+            defaultUrl={`/${this.props.serverNonComp.businessLineUrl}`}>
             <AppFrame>
               <AppSegment filledBackground>
                 {this.props.flash && <FlashAlerts flash={this.props.flash} />}
                 <div>
                   <PageRoute
                     exact
                     path="/:businessLineSlug/tasks/:taskId"
-                    title="Dispositions | Caseflow"
+                    title={`${appName} Dispositions | Caseflow`}
```
Comment (leikkisa) [inline client/app/nonComp/index.jsx]: Should we include these changes in a test somewhere?

**Comment by pkarman** [comment]
screen shot 2019-02-11 at 2 29 28 pm

File: client/app/nonComp/index.jsx (hunk 2 above)
Comment (pkarman) [reply client/app/nonComp/index.jsx]: ah yes, thanks @leikkisa - feature test added.

**Pull request merged**